samza-sql-shell/src/main/java/org/apache/samza/sql/client/cli/CliShell.java

```diff
@@ -265,564 +215,40 @@ private void printVersion() {
+        commandHandler = curCommandHandler;
+        LOG.info("Found commandHandler {} to handle command {}", commandHandler.getClass().getName(),
+            parsedCommand.getFullCommand());
         break;
-    }
-
-    List<String> lines = new ArrayList<>(minRowNeeded + 4);
-
-    // Header
-    StringBuilder line = new StringBuilder(terminalWidth);
-    line.append(CliConstants.SPACE);
-    line.append(HEADER_FIELD);
-    CliUtil.appendTo(line, seperatorPos - 1, CliConstants.SPACE);
-    line.append(SEPERATOR);
-    line.append(CliConstants.SPACE);
-    line.append(HEADER_TYPE);
-    lines.add(line.toString());
-    line = new StringBuilder(terminalWidth);
-    CliUtil.appendTo(line, longestLineCharNum - 1, LINE_SEP);
-    lines.add(line.toString());
-
-    // Body
-    AttributedStyle oddLineStyle = AttributedStyle.BOLD.foreground(AttributedStyle.BLUE);
-    AttributedStyle evenLineStyle = AttributedStyle.BOLD.foreground(AttributedStyle.CYAN);
-
-    final int fieldColSize = seperatorPos - 2;
-    final int typeColSize = terminalWidth - seperatorPos - 1 - 2;
-    for (int i = 0; i < rowCount; ++i) {
-      SqlSchema.SqlField sqlField = schema.getFields().get(i);
-      String field = sqlField.getFieldName();
-      String type = getFieldDisplayValue(sqlField.getFieldSchema());
-      int fieldLen = field.length();
-      int typeLen = type.length();
-      int fieldStartIdx = 0, typeStartIdx = 0;
-      while (fieldStartIdx < fieldLen || typeStartIdx < typeLen) {
-        line = new StringBuilder(terminalWidth);
-        line.append(CliConstants.SPACE);
-        int numToWrite = Math.min(fieldColSize, fieldLen - fieldStartIdx);
-        if (numToWrite > 0) {
-          line.append(field, fieldStartIdx, fieldStartIdx + numToWrite);
-          fieldStartIdx += numToWrite;
-        }
-        CliUtil.appendTo(line, seperatorPos - 1, CliConstants.SPACE);
-        line.append(SEPERATOR);
-        line.append(CliConstants.SPACE);
-
-        numToWrite = Math.min(typeColSize, typeLen - typeStartIdx);
-        if (numToWrite > 0) {
-          line.append(type, typeStartIdx, typeStartIdx + numToWrite);
-          typeStartIdx += numToWrite;
-        }
-
-        if (i % 2 == 0) {
-          AttributedStringBuilder attrBuilder = new AttributedStringBuilder().style(evenLineStyle);
-          attrBuilder.append(line.toString());
-          lines.add(attrBuilder.toAnsi());
-        } else {
-          AttributedStringBuilder attrBuilder = new AttributedStringBuilder().style(oddLineStyle);
-          attrBuilder.append(line.toString());
-          lines.add(attrBuilder.toAnsi());
-        }
       }
     }
-
-    // Footer
-    line = new StringBuilder(terminalWidth);
-    CliUtil.appendTo(line, longestLineCharNum - 1, LINE_SEP);
-    lines.add(line.toString());
-    return lines;
+    return new CommandAndHandler(parsedCommand, commandHandler);
   }
 
-  private String getFieldDisplayValue(SqlFieldSchema fieldSchema) {
```

Comment:
Shenoda - where did these changes go ? did we just delete them ?

Reply:
looks like lost update issue, created PR to restore them: https://github.com/apache/samza/pull/1114